pkg/config/config.go

```diff
@@ -35,14 +38,13 @@ type Config struct {
 
 // NewConfig is the constructor for Config.
 func NewConfig() *Config {
+	ctrl := Controllers{UNP: true}
 	return &Config{
 		LogLevel:        "info",
-		Controllers:     "node",
-		NodeWorkers:     1,
-		UNPWorkers:      1,
```

Comment:
So we don't want to spin up more than one worker?

Reply:
For now each resource being watched will get 1 thread. If that isn't enough in the future we can consider adding more workers, but for now it is OK.